### PR TITLE
Update non-core threejs dependencies

### PR DESCRIPTION
**Pull request by stianjensen**
Should have been updated when updating threejs to r71

### CODE REVIEW

**Comment by aleksanb** [comment]
Seems to break stuff in the demo.l

**Comment by stianjensen** [comment]
I'll do some more testing then

File: nin/dasBoot/lib/ConvolutionShader.js [hunk 1]
```diff
@@ -83,7 +83,7 @@ THREE.ConvolutionShader = {
 
 		values = new Array( kernelSize );
 		sum = 0.0;
-		for ( i = 0; i < kernelSize; ++i ) {
+		for ( i = 0; i < kernelSize; ++ i ) {
```
Comment (cristeahub) [inline nin/dasBoot/lib/ConvolutionShader.js]: :(

Comment (stianjensen) [inline nin/dasBoot/lib/ConvolutionShader.js]: I wouldn't have made that change personally. But this is threejs linting rules ^^

Comment (cristeahub) [inline nin/dasBoot/lib/ConvolutionShader.js]: Ye ^^

**Comment by sigvef** [comment]
Does this still break things?

**Comment by stianjensen** [comment]
Probably, since I haven't tested it more. I'll have another look. It would make sense that using the dependencies that came bundled with r71 works better than before, not worse :/

File: nin/dasBoot/lib/RenderPass.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ THREE.RenderPass = function ( scene, camera, overrideMaterial, clearColor, clear
 	this.oldClearAlpha = 1;
 
 	this.enabled = true;
-	this.clear = false;
+	this.clear = true;
```
Comment (stianjensen) [inline nin/dasBoot/lib/RenderPass.js]: error logs make it seem like this line might be causing it

**Comment by stianjensen** [comment]
Should be a-ok now

**Comment by sigvef** [comment]
The pixel ratio stuff in the effect composer might tank performance on hdpi laptops, but let's merge this anyway. We should update nin with a resolution selector or something down the line, so we can tackle the problem at that time.

**Pull request merged**